lstests/doc/README.md

```diff
@@ -0,0 +1,688 @@
+
+### `Service`
+
+A `Service` object represents the connection with the launcher service running in the background. Once a `Service` object is created, it will try to connect the launcher service automatically.
+
+After configuration, the `Service` will change the working directory, register (and possibly *modify*) the Logger, and then connect to launcher service. *Currently, it is only possible to connect to a local launcher service.*
+
+If there is already an existing launcher service running in the background, the `Service` object will by default connect to it (without starting a ew one). It is possible to override this default behavior by requesting to always start a new launcher service and/or to kill all the existing launcher service(s).
+
+A detailed explanation of the parameters to initialize a `Service` object can be found in its docstring.
+
+     Parameters
+     ----------
+    logger : Logger
+        Logger object which controls logging behavior.
+    addr : str
+        IP address of launcher service.
+        Currently, only local launcher service is supported. That is, only
+        default value "127.0.0.1" is supported.
+    port : int
+        Listening port of launcher service.
+        If there are multiple launcher service running in the background,
+        they must have different listening ports.
+        Default is 1234.
+    wdir : str
+        Working directory.
+        Default is the build folder.
+    file : str
+        Path to local launcher service file.
+        Can be either absolute or relative to the working directory.
+    gene : str
+        Path to the genesis file.
+        Can be either absolute or relative to the working directory.
+    start : bool
+        Always start a new launcher service.
+        Note that if to start a new instance alongside the existing ones,
+        make sure the listening ports are different. Otherwise, the
+        new launcher service will issue an error.
+        Default is False (will not start a new one if there is an existing
+        launcher service running in the background).
+    kill : bool
+        Kill existing launcher services (if any).
+        Kill all the existing launcher services running in the background
+        (and freshly start a new one).
+        Default is False (will not kill existing launcher services; instead
+        will connect to an existing launcher service).
+### `Cluster`
+
+A `Cluster` object represents a cluster of nodes running on launcher service. It is the major proxy for tests to communicate with launcher service. A `Cluster` object must have a `Service` object registered to it at initialization.
+
+ After configuration, the node cluster will be launched in one of two modes:
+
+1. BIOS mode, using `eosio.bios` to set producers;
+
+2. regular mode, using `eosio.token` to create and issue tokens, and using
```

Comment:
This mode should not exist as part of the new integration test framework. If a test really wants to simulate this kind of bootstrap, they will have to do so manually. But I really don't think any of our ported integration tests should continue using the `eosio.system` or `eosio.token` contracts.

Reply:
The regular launch mode has been removed from the source code. The `README.md` has been updated accordingly. Thank you.